articles/user-guide/host-programs.md

```diff
@@ -565,3 +565,11 @@ So, to run `MeasureSuperpositionArray`, we need to define a "wrapper" operation
 <img src="../media/hostprograms_jupyter_wrapper_def_sim_crop.png" alt="Wrapper function and simulate Jupyter cell" width="550">
 
 This operation can of course be used similarly with `%estimate` and other execution commands.
+
+### Visualizing Q# programs
+
+In a Q# Jupyter Notebook, you can visualize your Q# programs using the `%trace` magic command (more details about the command [here](https://docs.microsoft.com/qsharp/api/iqsharp-magic/trace)). Given a Q# operation as an argument, `%trace` will simulate that operation and trace out the gates in an execution path. This execution path is then rendered and output as a visualization within the Jupyter Notebook. For example, this is a visualization of a non-deterministic Q# program:
```

Comment:
```suggestion
In a Q# Jupyter Notebook, you can visualize your Q# programs using the `%trace` magic command (more details about the command [here](xref:microsoft.quantum.iqsharp.magic-ref.trace)). Given a Q# operation as an argument, `%trace` will simulate that operation and trace out the gates in an execution path. This execution path is then rendered and output as a visualization within the Jupyter Notebook. For example, this is a visualization of a non-deterministic Q# program:
```
The link gives me 404, I guess it is because it's a new command and until the next build it won't be generated. Anyways, the Python script that generates the IQ# articles also generates an `uid` for crossreferencing. A `xref` link is better suited here.

A warning will trigger if you commit this since the file won't be found. However, it will be found when the next build is run.

Reply:
Thanks for the feedback! Will make the change :)

Reply:
To clarify, I won't need to add the docs for `%trace` manually [here](https://github.com/MicrosoftDocs/quantum-docs-pr/tree/master/api/iqsharp-magic), right? It will get automatically generated in the next release?

Reply:
To avoid broken links on the main website, I think @anpaz-msft suggested offline targeting a feature branch on docs so that this doesn't accidentally get brought into live before it's ready.

Reply:
@cgranade is `feature/path_visualizer` the feature branch you made for this purpose?

Reply:
That's the one!